### DISCUSSION TITLE
Welcome to custompbx Discussions!

### DISCUSSION THREAD

**Discussion by zusrut** [custompbx/custompbx#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bradgarrison** [comment]
Nice job. This looks impressive.

**Reply by zusrut** [reply]
Thanks!

**Comment by dahilinet** [comment]
Hello friends.
I am new testing custompbx. And please tell me what is this error for.
2025/03/27 10:48:03 Reconnecting to ESL...
2025/03/27 10:48:13 Reconnecting to ESL...
2025/03/27 10:48:23 Reconnecting to ESL...
2025/03/27 10:48:33 Reconnecting to ESL...
2025/03/27 10:48:43 Reconnecting to ESL...
2025/03/27 10:48:53 Reconnecting to ESL...
 ESL2025/03/27 10:49:03 Reconnecting to ESL...
2025/03/27 10:49:13 Reconnecting to ESL...
2025/03/27 10:49:23 Reconnecting to ESL...
2025/03/27 10:49:33 Reconnecting to ESL...
2025/03/27 10:49:43 Reconnecting to ESL...

**Reply by zusrut** [reply]
Hello. It's mean custompbx cannot connect to freeswitch esl socket. Check your config and that fs is reachable.

**Reply by dahilinet** [reply]
Thank you for let me know. But i do not know how to set it up freeswitch esl socket. Any article ?

**Reply by dahilinet** [reply]
Ohh it is .fs_cli_conf   i found it and solved and it is working now, thank you for who made this perfect gui :)

**Comment by Tuguldur1024** [comment]
how do I login

**Reply by zusrut** [reply]
Hello. If all is running fine - default login and pass are: **admin** **admin**
